_data/packages.yml

```diff
@@ -1,3 +1,90 @@
+- package_name: astro-package3
```

Comment:
these are just dummy placeholder packages to test the **search for astropy** functionality